dist/styles/SoloRecipeView.css

```diff
@@ -93,12 +93,15 @@
 }
 
 .RecipeDescription {
```

Comment:
If we decided to use camelCase for out CSS classes, maybe do that here?

Reply:
im going to do a separate branch to change both of my css and jsx files

Reply:
👌